classes/class-ed-lti.php

```diff
@@ -475,20 +464,20 @@ private function lti_redirect_user_to_blog_without_login( $blog_id ) {
 	 * @param string $alphabet
 	 *
 	 * @return string
-	 * @throws Exception
-	 * @throws InvalidArgumentException
+	 * @throws \Exception
+	 * @throws \InvalidArgumentException
```

Comment:
Please change this to add a `use InvalidArgumentException` and remove the backslash